index.html

```diff
@@ -3821,37 +3820,36 @@ <h2>DID Resolution</h2>
 method-specific internal function to perform the actual <a>DID resolution</a>
 process. <a>DID resolver</a> implementations might implement and expose
 additional functions with different signatures in addition to the
-<code>resolve</code> function specified here.
+<code>resolveRepresentation</code> function specified here.
         </p>
 
         <section>
-            <h3>DID Resolution Input Metadata Properties</h3>
+            <h3>DID Resolution Options</h3>
 
             <p>
 The possible properties within this structure and their possible values are
 registered in the DID Specification Registries [[?DID-SPEC-REGISTRIES]]. This
-specification defines the following common properties.
+specification defines the following common properties for DID resolution
+options:
             </p>
 
             <dl>
                 <dt>
 accept
                 </dt>
                 <dd>
-The MIME type expressed as an <a data-lt="ascii string">ASCII string</a> of the
-caller's preferred <a>representation</a> of the <a>DID
-document</a>. The <a>DID resolver</a> implementation SHOULD use this value to
-determine the <a>representation</a> contained in the returned
-<code>did-document-stream</code> if such a <a>representation</a> is supported and
-available. This property is OPTIONAL. It is only used if the
-<code>resolveRepresentation</code> function is called and MUST be ignored if the
-<code>resolve</code> function is called.
+The MIME type of the caller's preferred <a>representation</a> of the <a>DID
+document</a>. The MIME type MUST be expressed as an  <a data-lt="ascii
+string">ASCII string</a>. The <a>DID resolver</a> implementation SHOULD use this
+value to determine the <a>representation</a> contained in the returned
+<code>didDocumentStream</code> if such a <a>representation</a> is supported and
+available. This property is OPTIONAL.
```

Comment:
Requesting a mime type makes no sense for an ADM resolution, which is why it was previously prohibited.

```suggestion
available. This property is OPTIONAL for the <code>resolveResolution</code> function
and MUST NOT be used with the <code>resolve</code> function.
```